calliope/preprocess/checks.py

```diff
@@ -650,35 +650,6 @@ def check_final(model_run):
                 "Time series `{}` contains non-unique timestamp values.".format(k)
             )
 
-    # Warn if loc/tech is defined in group constraint that doesn't exist in the model
```

Comment:
Why gone?

Reply:
Equivalent warnings come up in 'constraint_sets'